sdks/java/core/src/main/java/org/apache/beam/sdk/util/FluentBackoff.java

```diff
@@ -150,14 +149,20 @@ public FluentBackoff withMaxRetries(int maxRetries) {
         exponent, initialBackoff, maxBackoff, maxCumulativeBackoff, maxRetries);
   }
 
+  public String toString() {
+    return MoreObjects.toStringHelper(FluentBackoff.class)
```

Comment:
I wonder if it would be nice to include a line that is essentially "backoffs" (and just run out the maxRetries of the exponent). And/or provide a utility for that. And/or indicate what the total time spent backing off before failure would be (eg., when maxCumulativeBackoff isn't set, we would need to add up all the backoffs across the attempts).